crates/router/src/connector/airwallex/transformers.rs

```diff
@@ -142,6 +142,39 @@ impl<F, T> TryFrom<types::ResponseRouterData<F, AirwallexAuthUpdateResponse, T,
     }
 }
 
+#[derive(Default, Debug, Serialize, Eq, PartialEq)]
+pub struct AirwallexCompleteRequest {
+    request_id: String,
+    three_ds: AirwallexThreeDsData,
+    #[serde(rename = "type")]
+    three_ds_type: AirwallexThreeDsType,
+}
+
+#[derive(Default, Debug, Serialize, Eq, PartialEq)]
+pub struct AirwallexThreeDsData {
+    acs_response: Option<serde_json::Value>,
+}
+
+#[derive(Default, Debug, Serialize, Eq, PartialEq)]
+pub enum AirwallexThreeDsType {
+    #[default]
+    #[serde(rename = "3ds_continue")]
+    ThreeDSContinue,
+}
+
+impl TryFrom<&types::PaymentsCompleteAuthorizeRouterData> for AirwallexCompleteRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(item: &types::PaymentsCompleteAuthorizeRouterData) -> Result<Self, Self::Error> {
+        Ok(Self {
+            request_id: Uuid::new_v4().to_string(),
```

Comment:
Is this the reference id? we can pass attempt id from router data

Reply:
This is the unique id required by the connector for each request irrespective of its type.
So it will be unique and not payment_attempt_id